packages/grid/x-data-grid-pro/LICENSE

```diff
@@ -0,0 +1,12 @@
+Commercial License
+
+Copyright (c) 2020 Material-UI SAS
+
+MUI X Pro (https://mui.com/pricing/) is commercial software. You must purchase
+a license and agree to the End User License Agreement (EULA: https://mui.com/x/license/)
+to be able to use the software.
+
+Commercial licenses can be obtained at https://mui.com/store/items/material-ui-pro/.
```

Comment:
I want to rename this to https://mui.com/store/items/mui-x-pro/ but I'm hesitant, until #3135 is resolved

Reply:
I have renamed https://material-ui.com/store/contributors/material-ui/ and https://material-ui.com/store/contributors/material-ui-oss/ today.